docs/guide/user_guide.md

```diff
@@ -320,3 +320,89 @@ Once configured, you can interact with your Delta tables using natural language.
 - Specify any filters or conditions clearly
 - Use natural language to describe aggregations or calculations
 
+## Using the CDM MCP Server Deployed in Rancher2
```

Comment:
Why is rancher port forwarding required for this service but not, say, jupyterhub?